maker-core/delegates.md

```diff
@@ -1,5 +1,39 @@
+Delegates are ecosystem actors who operate through Protocol Delegation System smart contracts. MKR holders can delegate their voting power to them without the need to hand over the custody of their MKR.
+
+Constitutional Delegates (CDs) are one type of [Constitutional Conservers](https://endgame.makerdao.com/maker-core/tbc-governance/constitutional-conservers). They receive various benefits and are subject to specific requirements. They must put in place the governance strategies defined by the CVCs.
+
+CDs must not engage with other ecosystem actors or in operational activities. They are not allowed to compensate MKR holders that delegate to them. They must aim to provide governance information and research material to MKR holders.
+
+## Purpose
+
+The Constitutional Delegate role spares Delegates from having to create governance strategies. CDs must follow strategies that have been designed by CVC members.
+CDs do not take part in the creation of the strategies, they execute them. The risk of Delegates following their own agenda, since they are not required to hold any MKR, is then  eliminated.
+
+## Income
+CDs can receive compensation from the Maker Protocol. The compensation goes to the top 14 CDs that have the highest amount of combined MKR delegated to their delegate contracts and fulfill the participation requirements.
+
+There are two income levels for the top 14 qualified CDs: Prime and Reserve.  The 7 top qualified CDs are called the Prime Constitutional Delegates (PCDs). The top 8 to 14 qualified CDs are called the Reserve Constitutional Delegates (RCDs).
+
+Each income level has a monthly total compensation.
+Qualified CDs can earn an equal share of their income level total compensation. If a CD loses its qualification, their slot goes to the next qualified CD by the amount of delegated MKR.
+
+### Eligibility Requirements
+To get a share of the total compensation of their income level, CDs must fulfill the following requirements:
+1) Participation in CVC Subcommittee Meetings (monthly)
+    - PCDs must participate in 16+
+    - RDCs must participate in 4+
+2) Voting Activity (yearly)
+    - CDs must be active in 95%+ of all votes
+3) Communication Activity (yearly)
+    - CDs must communicate reasoning for their voting in 95%+ of all votes
+
+If these amounts are not reached, then CDs receive a reduced amount of the total CD income they are eligible for.
+
+## Visibility
+
+The EGF displays all CDs who follow two different CVC Governance Strategies. To do this, the CD must have two separate Protocol Delegation System smart contracts. Each of them is attached to the CVC whose Governance Strategy they follow. To calculate the income level, the total amounts of MKR delegated to both Governance Strategies of the CD are counted.
+
+When a [CVC](https://endgame.makerdao.com/maker-core/tbc-governance/cvc) bans a CD, the MKR holders decide through a vote if the CD can continue to be displayed with one Independent Governance Strategy.
```

Comment:
```suggestion
When a CVC bans a CD, the MKR holders decide through a vote if the CD can continue to be displayed with one Independent Governance Strategy.
```

Reply:
Also, "the MKR holders" - does this refer to only those who delegated to that CD or all MKR holders?

Reply:
It has been updated on MIP101. It is through a CVC decision.
"If a CD is banned by a CVC decision, the votes delegated for the CD to the strategy will instead count as a part of the delegates independent strategy."